components/brave_shields/browser/brave_shields_util_unittest.cc

```diff
@@ -701,90 +701,90 @@ TEST_F(BraveShieldsUtilTest, GetFingerprintingControlType_Default) {
   /* BLOCK */
   map->SetContentSettingCustomScope(
       ContentSettingsPattern::Wildcard(), ContentSettingsPattern::Wildcard(),
-      ContentSettingsType::PLUGINS, brave_shields::kFingerprinting,
+      ContentSettingsType::PLUGINS, brave_shields::kFingerprintingV2,
       CONTENT_SETTING_BLOCK);
   map->SetContentSettingCustomScope(
       ContentSettingsPattern::Wildcard(),
       ContentSettingsPattern::FromString("https://firstParty/*"),
-      ContentSettingsType::PLUGINS, brave_shields::kFingerprinting,
+      ContentSettingsType::PLUGINS, brave_shields::kFingerprintingV2,
       CONTENT_SETTING_BLOCK);
   setting = brave_shields::GetFingerprintingControlType(profile(), GURL());
   EXPECT_EQ(ControlType::BLOCK, setting);
   setting = brave_shields::GetFingerprintingControlType(
       profile(), GURL("http://brave.com"));
   EXPECT_EQ(ControlType::BLOCK, setting);
 
-  /* BLOCK_THIRD_PARTY */
+  /* DEFAULT */
   map->SetContentSettingCustomScope(
       ContentSettingsPattern::Wildcard(), ContentSettingsPattern::Wildcard(),
-      ContentSettingsType::PLUGINS, brave_shields::kFingerprinting,
+      ContentSettingsType::PLUGINS, brave_shields::kFingerprintingV2,
       CONTENT_SETTING_BLOCK);
   map->SetContentSettingCustomScope(
```

Comment:
this is not relevant anymore, there is no longer a first/third party scope. Can you please cleanup the tests in a followup?

Reply:
https://github.com/brave/brave-browser/issues/10078